HaxeManual/03-type-system.tex

```diff
@@ -503,7 +503,7 @@ \subsection{Import defaults / import.hx}
 \label{type-system-import-defaults}
 \since{3.3.0}
 
-This feature allows us to define default imports and usings that will be applied for all modules inside a directory, which is particularly handy for large code bases with a lot of helpers and static extensions because it reduces the amount of imports.
+This feature allows us to define default imports and using that will be applied for all modules inside a directory, which is particularly handy for large code bases with a lot of helpers and static extensions because it reduces the number of imports.
```

Comment:
Default import and usings – `usings` here refers to multiple `using` keywords, i.e. multiple static extensions.